Sources/StreamChat/Database/DTOs/PollDTO.swift

```diff
@@ -23,11 +23,20 @@ class PollDTO: NSManagedObject {
     @NSManaged var maxVotesAllowed: NSNumber?
     @NSManaged var votingVisibility: String?
     @NSManaged var createdBy: UserDTO?
-    @NSManaged var latestAnswers: Set<PollVoteDTO>
```

Comment:
why do we change this? Could it have impact on the existing customers in production?

Reply:
No because the models remain the same. This property and ownVotes conflict with each other because they are the same entity. So we need to merge both properties in one. CoreData only knows about votes that are linked to a Poll, it does not care if it is an answer or if it is a vote from the current user